demo/sections/index.ts

```diff
@@ -4,6 +4,7 @@ import { layouts } from '@/demo/sections/layouts'
 
 export const sections: Section = {
   components,
+  designTokens: () => import('./DesignTokens.vue'),
```

Comment:
Seems like the dynamic menu stuff isn't expecting camel case stuff here. Ugly but I think this will fix it
<img width="273" alt="image" src="https://github.com/PrefectHQ/prefect-design/assets/6200442/61ed580c-ed2e-4278-9ec8-1310fab34c09">

```suggestion
 'Design Tokens': () => import('./DesignTokens.vue'),
```